src/compile/mark/mixins.ts

```diff
@@ -36,7 +36,7 @@ export function markDefProperties(mark: MarkDef, props: (keyof MarkDef)[]) {
   }, {});
 }
 
-export function valueIfDefined(prop: string, value: VgValueRef): VgEncodeEntry {
+export function valueIfDefined(prop: string, value: string): VgEncodeEntry {
```

Comment:
It's not needed. I can add them but we can also add it when ts complains. VgValueRef was definitely wrong.